infra/build/functions/oss_fuzz_on_demand.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#!/usr/bin/env python3
+"""Starts and runs coverage build on Google Cloud Builder.
+
+Usage: build_and_run_coverage.py <project>.
```

Comment:
A lot of unused pieces from this seem to still be in this file. Can you please clean this up?